src/components/MAPI/clusters/ClusterDetail/ClusterDetailWidgetProvider.tsx

```diff
@@ -138,65 +143,97 @@ const ClusterDetailWidgetProvider: React.FC<
-      {...props}
-    >
-      <OptionalValue value={getClusterRegionLabel(cluster)} loaderWidth={85}>
-        {(value) => <Text>{value}</Text>}
-      </OptionalValue>
-      <OptionalValue value={region} loaderWidth={80}>
-        {(value) => (
-          <Text>
-            <code>{value}</code>
-          </Text>
-        )}
-      </OptionalValue>
-      <StyledDot />
-      <OptionalValue value={getClusterAccountIDLabel(cluster)}>
-        {(value) => <Text>{value}</Text>}
-      </OptionalValue>
-      <OptionalValue
-        value={accountID}
-        loaderWidth={300}
-        replaceEmptyValue={false}
-      >
-        {(value) =>
-          value === '' ? (
-            <NotAvailable />
-          ) : (
-            <StyledLink
-              color='text-weak'
-              href={accountIDPath}
-              rel='noopener noreferrer'
-              target='_blank'
-            >
+    <ClusterDetailWidget title='Provider' inline={true} {...props}>
+      <Box direction='row' data-testid='provider-group'>
+        <GroupLabel size={provider === 'aws' ? 'small' : 'medium'}>
+          <OptionalValue
+            loaderWidth={85}
+            value={getClusterRegionLabel(cluster)}
+          >
+            {(value) => <Text>{value}</Text>}
+          </OptionalValue>
+        </GroupLabel>
+        <OptionalValue value={region}>
+          {(value) => (
+            <Text>
               <code>{value}</code>
-              <i
-                className='fa fa-open-in-new'
-                aria-hidden={true}
-                role='presentation'
-              />
-            </StyledLink>
-          )
-        }
-      </OptionalValue>
+            </Text>
+          )}
+        </OptionalValue>
+      </Box>
+      <Box direction='row' data-testid='provider-group'>
+        <GroupLabel size={provider === 'aws' ? 'small' : 'medium'}>
+          <OptionalValue
+            loaderWidth={85}
+            value={getClusterAccountIDLabel(cluster)}
+          >
+            {(value) => <Text>{value}</Text>}
+          </OptionalValue>
+        </GroupLabel>
+        <OptionalValue
+          value={accountID}
+          loaderWidth={200}
+          replaceEmptyValue={false}
+        >
+          {(value) =>
+            value === '' ? (
+              <NotAvailable />
+            ) : accountIDPath === '' ? (
+              <code>{value}</code>
+            ) : (
+              <StyledLink
+                color='text-weak'
+                href={accountIDPath}
+                rel='noopener noreferrer'
+                target='_blank'
+              >
+                <code>{value}</code>
+                <i
+                  className='fa fa-open-in-new'
+                  aria-hidden={true}
+                  role='presentation'
+                />
+              </StyledLink>
+            )
+          }
+        </OptionalValue>
+      </Box>
+      {provider === 'azure' && (
+        <Box direction='row' data-testid='provider-group'>
```

Comment:
```suggestion
      </Box>
      {provider === Providers.AZURE && (
        <Box direction='row' data-testid='provider-group'>
```
Could we use the constant instead of a string literal here?